.github/renovate.json

```diff
@@ -1,4 +1,17 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": ["github>cgrindel/renovate_presets"]
+  "extends": [
+    ":dependencyDashboard",
+    ":semanticPrefixFixDepsChoreOthers",
+    "group:monorepos",
+    "group:recommended",
+    "replacements:all",
+    "workarounds:all"
+  ],
+  "packageRules": [
+    {
+      "matchFiles": ["MODULE.bazel"],
+      "enabled": false
```

Comment:
Was `"enabled": false` intended here?

Reply:
Yes. We do not want to automatically update the ruleset's dependencies automatically.

[Related discussion](https://github.com/bazel-contrib/SIG-rules-authors/discussions/82)